docs/integrations-in-rancher/logging/custom-resource-configuration/outputs-and-clusteroutputs.md

```diff
@@ -2,7 +2,9 @@
 title: Outputs and ClusterOutputs
 ---
 
-For the full details on configuring `Outputs` and `ClusterOutputs`, see the [Banzai Cloud Logging operator documentation.](https://banzaicloud.com/docs/one-eye/logging-operator/configuration/output/)
+For the full details on configuring `Outputs` and `ClusterOutputs`, see the [Banzai Cloud Logging operator documentation.](https://banzaicloud.com/docs/one-eye/logging-operator/configuration/output/).
+
+For memory problems with the logging buffer, see [Rancher Integration with Logging Services: Troubleshooting](../pages-for-subheaders/logging.md#Troubleshooting).
```

Comment:
Are there plans to add more entries around memory issues with the buffer under `Troubleshooting`? If not, we could link to the subsection directly.

Reply:
Updated the links to point to the subsection